api/turing/middleware/openapi_validation.go

```diff
@@ -46,10 +50,26 @@ func NewOpenAPIV2Validation(swaggerYamlFile string, options OpenAPIValidationOpt
 	if err != nil {
 		return nil, err
 	}
+	var routers []*openapi3filter.Router
+	routers = append(
+		routers,
+		openapi3filter.NewRouter().WithSwagger(v3Swagger),
+	)
```

Comment:
`routers` can be defined and initialised in a single line
```go
routers := []*openapi3filter.Router{openapi3filter.NewRouter().WithSwagger(v3Swagger)}
```

Reply:
Done